Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -562,7 +562,6 @@ public ElasticsManager ElasticsManager
         private static readonly List<Vector3> TotalBoundsCorners = new List<Vector3>();
 
         private Vector3[] boundsCorners = new Vector3[8];
-        public Vector3[] BoundsCorners { get; private set; }
```

Comment:
Can we `Obsolete` this instead of fully removing? Especially if we're shipping this in 2.7.3

Reply:
and/or is there value in updating this to return `boundsCorners`?

Reply:
Okay, can do.